enemy-data/mechanics/enemy-attributes.md

```diff
@@ -28,6 +28,13 @@ Monster skill effects can be taunted. In general, these skill effects target the
 
 The bubbles spawned can be effectively iframed by swapping characters at the precise moment. However, the hydro bubble will persist on the character that was initially hit for approximately 5 seconds. For more details see: [Mist Bubble iFrame](../../evidence/enemy-data/enemy-attributes.md#mist-bubble-iframe)
 
+## Targeting Ignores Enemy Hitbox Location
+
+**By:** Kourinn\#6001  
```

Comment:
no need to mention the ticket author here if you're already linking the vault entry